### ISSUE TITLE
Changelog page on Flowershow website

### ISSUE THREAD

**Issue by olayway** [flowershow/flowershow#39]
It would be nice to display the changelog file on the Flowershow website. It's good to show users that the product is under active development.
A great example from Penpot: https://penpot.app/dev-diaries.html.
Another great example from GitHub: https://github.blog/changelog/

https://linear.app/changelog is really good (@rufuspollock 2026-01-15)

<img width="1122" height="906" alt="Image" src="https://github.com/user-attachments/assets/6eeae3db-a77b-431d-8a21-aef44684d229" />

## 2026-02-25

https://rufuspollock.com/z/2026-02-25-changelogs-lightweight-best-practice

did some research on changelog setup - here's some thoughts. basic point is we use CHANGELOG.md. But we could auto generate from commit logs using git cliff but i suspect this needs a lot of discipline and good changelogs look a bit better cf https://linear.app/changelog - so i think something agentic (a skill) which can generate when we need ... (i also think a changelog entry per big feature plus smaller changelogs with bundles works well)

## 2026-01-28

Implementation suggestions as of today would be:
1. we just have a folder called changelog.
2. each changelog item is a markdown file in that folder. can be short or long. it's just the name of the feature and anything else needed. screenshots are a must!
3. We show those on the changelog page just using the list/bases feature.

That would be, I think, just the absolute simple minimum viable setup.

**Comment by rufuspollock** [comment]
WONTFIX. Nice to have and maybe we'll come back to it someday.

**Comment by rufuspollock** [comment]
I think this would be cool - could just be a list of blog posts for now using Base support 😄

**Comment by rufuspollock** [comment]
I'm upping this to 4🔥 😄

**Comment by Abeelha** [comment]
## Changelog candidates: Oct 2025 – Mar 2026

### ~~PR here: https://github.com/flowershow/flowershow/pull/1139 | for mcp-changelogs~~ CLOSED

Scanned recent commits and feature work. Here are 5–10 items worth writing up as proper changelog entries, ordered roughly by user impact:

1. **Obsidian Bases support (BETA)** — Nov 2025 — Render Obsidian Bases database queries (tables, cards, lists) live on your published site. First-class Dataview-style querying right in the browser. (`feat: Bases support BETA #741`)

2. **MCP Server** — Feb 2026 — An MCP server that lets AI assistants (Claude, Cursor, ChatGPT) publish notes directly to your Flowershow site using natural language. (`feat: basic MCP Server #1054`)

3. **Google Sign-in** — Jan 2026 — Sign in with Google in addition to GitHub. Removes the GitHub requirement for new users. (`feat: Google auth option and switch to GitHub App for GitHub auth #996`)

4. **Personal Access Tokens (PAT) + API** — Jan 2026 — Create long-lived tokens for headless publishing from the CLI, scripts, and automation tools. (`feat!: PAT tokens support and API improvements #1028`)

5. **Navbar dropdown menus** — Feb 2026 — Add one-level dropdown menus to your site navigation in `config.json`. (`feat: add one-level dropdown menu support to navbar`)

6. **Customizable footer** — Jan 2026 — Define your own footer links and content in `config.json`. (`feat: customizable footer`)

7. **Audio and video embeds** — Nov 2025 — Embed audio and video files using Obsidian wiki-link syntax (`![[audio.mp3]]`). (`feat: support audio and video embeds`)

8. **Regular Markdown mode** — Oct/Nov 2025 — Choose plain Markdown rendering (no MDX) for simpler, faster pages and full parity with how Obsidian renders notes. (`feat: MD/MDX rendering mode #736`)

9. **Drag-and-drop multi-file publishing** — Jan 2026 — Drop up to 5 files on the homepage for instant publishing with no account required. (`feat: drag n drop up to 5 files on the new landing page`)

10. **Text highlighting with `==text==`** — Oct 2025 — Obsidian-style highlight syntax now renders on published sites. (`feat: text highlighting with ==text==`)

---

**Recommended for full write-ups (linear.app style):**

- 🏆 **MCP Server** — Most timely and differentiating right now. The story is compelling: "publish to your Flowershow site from Claude, Cursor, or any AI assistant without leaving your editor." Writing this up.
- 🏅 **Obsidian Bases** — The deepest feature, most unique to Flowershow. A proper write-up would showcase Flowershow as the only hosted Obsidian publisher that renders live Bases queries. Worth doing next.

Writing up the MCP Server entry now as a PR.

**Comment by Abeelha** [comment]
## TLDR

### **10 changelog candidates (Oct 2025 – Mar 2026):**
-   1. Obsidian Bases BETA — render live database queries on published sites
-    ~~2. MCP Server — publish from Claude/Cursor/ChatGPT ← recommended write-up #1~~ -not released
-   3. Google Sign-in — no GitHub account needed
-   4. PAT Tokens + API — headless publishing and automation
-   5. Navbar dropdown menus — one-level dropdowns in config.json
-   6. Customizable footer — define footer links in config.json
-   7. Audio/video embeds — ![[audio.mp3]] Obsidian syntax
-   8. Regular Markdown mode — plain MD rendering, no MDX
-   9. Drag-and-drop multi-file publishing — up to 5 files on homepage
-   10. Text highlighting ==text== — Obsidian-style highlights

cc. @rufuspollock

**Comment by rufuspollock** [comment]
@Abeelha  i would leave mcp server for now - though it could be soon. focus on:

1, 3, 4 (and doc cli from autumn separately), 5, 6, 8 and 9

**Comment by Abeelha** [comment]
## Updated status ~~(MCP removed — not shipped yet)~~

**Written up and in PR:**
- ✅ **Obsidian Bases BETA** (`2025-11-25-obsidian-bases.md`) — render live database queries as tables, cards, and lists on your published site
- ✅ **Sidebar improvements** (`2026-03-04-sidebar-navigation.md`) — on by default, mobile slide-out drawer, route-specific visibility (adapted from blog post)

**Still to write up (confirmed shipped, in roughly priority order):**
1. **Google Sign-in** — Jan 2026 — no GitHub account needed (`#996`)
2. **PAT Tokens + API** — Jan 2026 — headless publishing and automation (`#1028`)
3. **Customizable footer** — Jan 2026 — define footer links in `config.json`
4. **Navbar dropdown menus** — Feb 2026 — one-level dropdowns in `config.json`
5. **Regular Markdown mode** — Oct/Nov 2025 — plain MD rendering, no MDX (`#736`)
6. **Drag-and-drop multi-file publishing** — Jan 2026 — up to 5 files on homepage
7. **CLI publishing tool** — Autumn 2025 — separate entry covering the CLI

_**MCP Server — hold** — feature exists in the branch but not confirmed shipped to production yet. Will write up once live._

PR: https://github.com/flowershow/flowershow/pull/1145 (branch `changelog/obsidian-bases-sidebar`)
cc. @rufuspollock

**Comment by rufuspollock** [comment]
FIXED. https://flowershow.app/changelog and https://flowershow.app/changelog/2026-02-21-changelog-page-launched

<img width="1453" height="1029" alt="Image" src="https://github.com/user-attachments/assets/a4ca4916-c07f-4be4-9479-2568688c1480" />

**Comment by rufuspollock** [comment]
@Abeelha i've closed this but i think you can split out a separate issue assigned to you for doing the addition of changelog pages - and copy over your info on your shortlist and my comments.

I also note you could do an entry on "obsidian compatibility" that compiles all the main items from this milestone/view https://github.com/orgs/flowershow/projects/1/views/23
